### PR TITLE
Not exiting when executable not found

### PR DESCRIPTION
**Pull request by yoavrotems**
Regrading https://github.com/aquasecurity/kube-bench/issues/701 where kube bench is crushing when not finding components

### CODE REVIEW

File: cmd/common.go [hunk 1]
```diff
@@ -91,7 +91,7 @@ func runChecks(nodetype check.NodeType, testYamlFile string) {
 
 	// Checks that the executables we need for the section are running.
 	if err != nil {
-		exitWithError(fmt.Errorf("failed to get a set of executables needed for tests: %v", err))
+		glog.V(1).Info(fmt.Sprintf("failed to get a set of executables needed for tests: %v", err))
```
Comment (lizrice) [inline cmd/common.go:94]: Can we still just continue with running the tests in this section? Or should we be returning from this function with this error? I guess we should be returning [FAIL] or [WARN] for each test in this section

**Review by lizrice — CHANGES_REQUESTED** [review]
It's only one line change, but it could have huge consequences! This would be a good example where you can help the reviewer by providing the output before and after the change, using an example where a section fails

File: cmd/common.go (hunk 1 above)
Comment (yoavrotems) [reply cmd/common.go:94]: Its before running anything, its changing the behavior from not running and run and fail/warn on the relate tests 
This is the output before the change: 
image
After the change we can see that now we can run and get cis output results:
image

**Review by lizrice — APPROVED** [review]
OK, I'm convinced by seeing the output change!

**Pull request merged**